.fixtures.yml

```diff
@@ -1,6 +1,12 @@
 fixtures:
   repositories:
     apt: "git://github.com/puppetlabs/puppetlabs-apt.git"
+    cron_core:
```

Comment:
are those modules new requirements ? if so, please add them to the `metadata.json`

Reply:
Not really, Puppet 6.x has removed some built-in modules which are now provided as third-party modules.

However, the puppet-agent package ships with these modules, so you just have to deal with them in the test suite like here.